src/config/contractInfo.js

```diff
@@ -1,29 +1,29 @@
 import dotenv from 'dotenv';
+import Joi from 'joi';
 
 dotenv.config();
 
-const isFilecoinProduction = process.env.FILECOIN_PRODUCTION === 'true';
```

Comment:
As @mariacarmina said above, we need this `FILECOIN_PRODUCTION` env so that we can turn off testnets.

Reply:
The problem i see with this is we dont know how many testnets and mainnets does the team plans to support?
with the current config only the list of chainIDs changes for testnet or main-nets 
@jamiehewitt15 @mariacarmina

Reply:
Our Approach requires no hardcoding whatsoever for addresses or RPC_URL